pkg/build/util/util.go

```diff
@@ -74,3 +75,9 @@ func NameFromImageStream(namespace string, ref *kapi.ObjectReference, tag string
 func IsBuildComplete(build *buildapi.Build) bool {
 	return build.Status.Phase != buildapi.BuildPhaseRunning && build.Status.Phase != buildapi.BuildPhasePending && build.Status.Phase != buildapi.BuildPhaseNew
 }
+
+// ConfigSelector returns a label Selector which can be used to find all
+// builds for a BuildConfig.
+func ConfigSelector(name string) labels.Selector {
+	return labels.Set{buildapi.BuildConfigLabel: name}.AsSelector()
```

Comment:
>should have been a part of your pull introducing the new label, sorry i didn't think of it at the time. can you do a follow up?

@bparees Ok, I'll update code. Am I right that I just need to add new commit to the existing branch and reopen original ticket? Or it will be a completely new PR?